tests/namespace/test_utilities_namespace.py

```diff
@@ -0,0 +1,81 @@
+import importlib
+import pathlib
+
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+namesapce_data = pathlib.Path(__file__).parent / 'namespace-utilities.txt'
+with open(namesapce_data) as f:
```

Comment:
`namesapce_data` -> `namespace_data`

Reply:
FYI you can leave specific code suggestions on github :) https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request

![where the code suggestion icon is](https://docs.github.com/assets/cb-26290/mw-1440/images/help/pull_requests/suggestion-block.webp)